aviary/aviary.py

```diff
@@ -195,6 +195,23 @@ def main():
         default=" "
     )
 
+    base_group.add_argument(
+        '--snakemake-profile',
+        help='Snakemake profile (see https://snakemake.readthedocs.io/en/stable/executing/cli.html#profiles)\n'
+             'Create profile as `~/.config/snakemake/[CLUSTER_PROFILE]/config.yaml`. \n'
```

Comment:
Unclear whether you mean the user or aviary does the creating. I think an example yml in the doco would go a long way here.

Reply:
I added some stuff under Advanced Usage. Is that what you wanted?